packages/mds-ingest-service/service/provider.ts

```diff
@@ -155,6 +157,17 @@ export const IngestServiceProvider: ServiceProvider<IngestService, IngestService
     }
   },
 
+  getH3Bins: async (context, params: GetH3BinOptions) => {
+    try {
+      const { start, end = now(), h3_resolution, k = K_HOURLY } = params
+      return ServiceResult(await IngestRepository.getH3Bins(validateGetH3BinsOptions({ start, end, h3_resolution, k })))
```

Comment:
Looks like we do this destructuring & defaulting twice (it's also done in the repo method), could we just pass in `params` here to the validator, and then do the defaulting only in the repo method?

Reply:
Oh thanks, meant to delete the defaulting in the repo method.